composer.json

```diff
@@ -27,28 +27,27 @@
             "echo \"Updating Resource Server Dependencies\" && cd resource-server && composer update"
         ],
         "test": [
-            "composer update-okta-oidc-tck",
```

Comment:
Why did you remove the update-okta-oidc-tck?

Reply:
The script `setup-env.js` updates the TCK when it's run